configure-clusters/src/main/java/com/esri/arcgismaps/sample/configureclusters/screens/MainScreen.kt

```diff
@@ -0,0 +1,413 @@
+                            Divider()
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+/**
+ * Composable function to display the cluster max scale controls within the cluster controls bottom
+ * sheet.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun ClusterMaxScaleControls(
+    clusterMaxScaleOptions: List<Int>,
+    clusterMaxScale: Int,
+    updateClusterMaxScale: (Int) -> Unit
+) {
+    Row(verticalAlignment = Alignment.CenterVertically) {
+        Text(
+            "Cluster max scale",
+            modifier = Modifier.padding(8.dp)
+        )
+        var expanded by rememberSaveable { mutableStateOf(false) }
+        Box(
+            modifier = Modifier
+                .wrapContentHeight()
+                .wrapContentWidth()
+                .wrapContentSize(Alignment.TopStart)
+                .padding(8.dp)
+        ) {
+            ExposedDropdownMenuBox(
+                expanded = expanded,
+                onExpandedChange = { expanded = !expanded }
+            ) {
+                TextField(
+                    value = clusterMaxScale.toString(),
+                    onValueChange = {},
+                    readOnly = true,
+                    trailingIcon = { ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded) },
+                    modifier = Modifier.menuAnchor()
+                )
+                ExposedDropdownMenu(
+                    expanded = expanded,
+                    onDismissRequest = { expanded = false }
+                ) {
+                    clusterMaxScaleOptions.forEachIndexed { index, clusterRadius ->
+                        DropdownMenuItem(
+                            text = { Text(clusterRadius.toString()) },
+                            onClick = {
+                                updateClusterMaxScale(
+                                    index
+                                )
+                                expanded = false
+                            })
+                        // show a divider between dropdown menu options
+                        if (index < clusterMaxScaleOptions.lastIndex) {
+                            Divider()
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+/**
+ * Composable function to display the cluster info content from the pop up within a bottom sheet.
+
+ */
+@Composable
+private fun ClusterInfoContent(
+    popUpTitle: String,
+    popUpInfo: Map<String, Any?>,
+    onDismiss: () -> Unit
+) {
+    Column(Modifier.background(MaterialTheme.colorScheme.background)) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Text(
+                modifier = Modifier
+                    .padding(horizontal = 30.dp, vertical = 12.dp)
+                    .weight(6f),
```

Comment:
Instead of defining `weight` Modifers here to set the horizontal spacing, I believe a better approach would be to use `horizontalArrangement` type and set the content padding on the Row here: 
```suggestion
        Row(
            modifier = Modifier.fillMaxWidth().padding(horizontal = 30.dp, vertical = 12.dp),
            verticalAlignment = Alignment.CenterVertically,
            horizontalArrangement = Arrangement.SpaceBetween
        ) {
            Text(
```